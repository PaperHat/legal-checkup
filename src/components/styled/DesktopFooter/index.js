import styled from 'styled-components'

const StyledDesktopFooter = styled.div`
  width: 100%;
  height: 100px;
  background-color: #202020;
  line-height: 100px;
  ${'' /* flex-direction: row;
  align-items: center;
  display: flex;
  justify-content: justify-content; */} position: absolute;
  bottom: 0px;
`
StyledDesktopFooter.displayName = 'StyledDesktopFooter'

export default StyledDesktopFooter
