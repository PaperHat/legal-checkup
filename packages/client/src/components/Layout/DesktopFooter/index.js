import { PRIVACY_POLICY_PATH, TERMS_OF_USE_PATH, COMMUNITY_LAW_PATH } from '../../../state/constants'

import React from 'react'
import styled from 'styled-components'
import DesktopFooterLinkExternal from '../../Link/DesktopFooterLinkExternal'
import DesktopFooterLinkInternal from '../../Link/DesktopFooterLinkInternal'

const StyledDesktopFooter = styled.div`
  width: 100%;
  height: 100px;
  background-color: #202020;
  line-height: 100px;
  position: absolute;
  bottom: 0px;
`
StyledDesktopFooter.displayName = 'StyledDesktopFooter'

export default function DesktopFooter () {
  return (
    <StyledDesktopFooter>
      <DesktopFooterLinkExternal href={COMMUNITY_LAW_PATH} target='_blank'>
        Â© 2018 by Community Law Wellington and Hutt Valley
      </DesktopFooterLinkExternal>
      <DesktopFooterLinkInternal href={PRIVACY_POLICY_PATH}>Privacy Policy</DesktopFooterLinkInternal>
      <DesktopFooterLinkInternal href={TERMS_OF_USE_PATH}>Terms of Use</DesktopFooterLinkInternal>
    </StyledDesktopFooter>
  )
}
