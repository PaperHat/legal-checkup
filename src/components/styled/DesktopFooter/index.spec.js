import React from 'react'
import { shallow } from 'enzyme'

import DesktopFooter from '.'

describe('components:styled:DesktopFooter', function () {
  it('renders the DesktopFooter and CSS properties properly', () =>
    expect(toJson(shallow(<DesktopFooter />))).toMatchSnapshot())
})
