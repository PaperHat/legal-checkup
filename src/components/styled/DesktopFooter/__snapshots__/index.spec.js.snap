// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:DesktopFooter renders the DesktopFooter and CSS properties properly 1`] = `
.c0 {
  width: 100%;
  height: 100px;
  background-color: #202020;
  line-height: 100px;
  position: absolute;
  bottom: 0px;
}

<div
  className="c0"
/>
`;
